components/elasticsearch/scripts/create.py

```diff
@@ -238,6 +238,24 @@ def _install_elasticsearch():
     utils.systemd.enable(ES_SERVICE_NAME, append_prefix=False)
 
 
+def _wait_for_shards(port, ip):
+    ctx.logger.info('Waiting for shards to be active...')
```

Comment:
please add a docstring explaining the reason behind this. it's not obvious why we need this.